src/MongoDB.Driver.Core/Core/Compression/ICompressor.cs

```diff
@@ -39,7 +40,7 @@ public enum CompressorType
         /// <summary>
         /// The content of the message is compressed using zstandard. 
         /// </summary>
-        ZStandard = 3
+        Zstd = 3
```

Comment:
As I commented in the ticket:

    We use "ZStandard" in our code.

    Rather than changing the enum I would change how the enum values are mapped when written to the wire protocol.

Note that using `Enum.Parse` and `someEnumValue.ToString` is FRAGILE because it ties the external string representation to the **NAME** of the C# enum value.

It is not unusual to refactor the name of an enum value, and that should NOT create a bug. In fact, this bug was probably created by such an innocent refactoring in the first place.

A more ROBUST approach is to have mapping functions from/to the C# enum value and the external string representation and thus decouple them.

Reply:
Dmitry is proposing the simplest possible change.

If we accept the FRAGILITY that comes with linking the C# name with the external string we can do this.

This wouldn't be the only place we have such a fragile linking.

What do others think?

Reply:
Isn't it a BC break to change the name of an enum value like this?

Reply:
Good point. I hadn't thought of that.

This is a public type.

Reply:
oh, it's true, in this case, we don't have a choice, I will add a mapping then

Reply:
done